src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Participants/ParticipantsControllerTestsHelper.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Equinor.ProCoSys.IPO.WebApi.IntegrationTests.Participants
+{
+    public static class ParticipantsControllerTestsHelper
+    {
+        private const string Route = "Participants";
```

Comment:
Route is not in use. I now see that Routes in Controllers are created in a strange manner. I'll talk to @bratli about this